### DISCUSSION TITLE
Biome rtp

### DISCUSSION THREAD

**Discussion by thewickening** [Zrips/CMI#8620] [Enhancement]
I know biomertp has this feature but to be honest, I refuse to use any rtp plugin but cmi as it works perfectly with 0 tps drops with 80+ people on rtping.

I would love a biome rtp system.

**Comment by Zrips** [comment]
Can you elaborate more on this request? Are you talking about teleportation to specific preset biomes from config file?

**Reply by thewickening** [reply]
Yeah if possible, how the rtp system currently can generate the worlds if enabled, if it could do the same for biomes, then you can individually turn each one off or on to your liking.

As well as having separate cooldowns for biome rtp and permissions.

I hope that explained it a bit better :)
